### ISSUE TITLE
Support Apple Silicon

### ISSUE THREAD

**Issue by ghost** [Iconscout/iconscout-app#3]
I want you to make a native app that runs on M1 MacBook

**Comment by romanr** [comment]
Yes please the current app 1.0.3 exits immediately after launch on M1. 
Rosetta installed and other old apps work.
